packages/dds/tree/src/feature-libraries/modular-schema/fieldChangeHandler.ts

```diff
@@ -190,7 +210,7 @@ export interface FieldEditor<TChangeset> {
  * The `index` represents the index of the child node in the input context.
  * The `index` should be `undefined` iff the child node does not exist in the input context (e.g., an inserted node).
  */
-export type ToDelta = (child: NodeId) => DeltaFieldMap;
+export type ToDelta = (child: NodeId) => DeltaFieldMap | undefined;
```

Comment:
Does this need to be changed as part of this PR? What was the motivation behind this change?

Reply:
It definitely does not. I just saw a todo for it and thought it would be simple enough to address.

Reply:
It's nice of you to try to leave the place cleaner than you found it, but it's best to keep that kind of cleanup in separate PRs.

I see you reverted the change, so I'm resolving this thread.